docs/toggleable-passes.md

```diff
@@ -146,3 +147,29 @@ main = (a a)
 @id = (a a)
 @main = @id
 ```
+
+## Cross-refs
+
+Is triggered only if the [pre-reduce](#pre-reduce) option is enabled.
+
+If enabled, will dereference and try to find the smallest net.
```

Comment:
"When enabled, the compile-time reduction pass will also reduce reference applications, leading to full normalization of the compiled net of each function."
Something like this, explaining with a little bit more details